README.md

```diff
@@ -390,9 +390,9 @@ To customize the time delay between successive in-app messages (default value of
 
 If you are already using in-app messages, then you will have to make the following changes to your code:
 
-1. `spawnInAppNotification` is no longer needed. In-app messages will be displayed automatically. Remove all `spawnInAppNotification` calls.
-2. Stop polling for in-app messages. The SDK will issue a callback using `config.inAppDelegate` [as explained above](#overriding-whether-to-show-or-skip-a-particular-in-app-message).
-3. Remove calls to 'IterableAPI.getInAppMessages()' and use `IterableAPI.inAppManager.getInAppMessages()` instead.
+1. `spawnInAppNotification` is no longer needed and will fail to compile. In-app messages are now displayed automatically.
+2. Stop polling for in-app messages using a timer etc. The SDK will issue a callback using `config.inAppDelegate` [as explained above](#overriding-whether-to-show-or-skip-a-particular-in-app-message).
+3. If you want to handle all in-app messages manually, as before these changes, define an `inAppHandler` on `IterableConfig` that always returns `InAppShowResponse.skip` to never show in-app messages automatically, and use `getInAppManager().getMessages()` to get the messages and `getInAppManager().showMessage(message)` to show a specific message.
```

Comment:
```suggestion
3. If you want to handle all in-app messages manually, as before these changes, define an `inAppHandler` on `IterableConfig`. It should return `InAppShowResponse.skip` to prevent in-app messages from showing automatically, or it can call `getInAppManager().getMessages()` to get the messages and `getInAppManager().showMessage(message)` to show a specific message.
```

Reply:
This looks good although 'or it can call `getInApp...`' is not right here. It is more of an 'and'. So just getting rid of 'or' should be good.

Reply:
I modified your  suggestion a little.